setup.cfg

```diff
@@ -96,7 +96,7 @@ tests_require =
     pytest-asyncio ; python_version >= '3.7'
     pytest-mock ; python_version >= '3.7'
     httpx ; python_version >= '3.6'
-
+    sanic ; python_version >= '3.6'
```

Comment:
Should this be `>= '3.7'` instead? In the jenkins exclude you have excluded python 3.6.

Reply:
Yes, that is correct. Sanic does not support 3.6